typescript/cli/src/deploy/utils.ts

```diff
@@ -55,12 +55,13 @@ export async function runPreflightChecksForChains({
   assertSigner(signer);
   logGreen('✅ Signer is valid');
 
-  await nativeBalancesAreSufficient(
-    multiProvider,
-    signer,
-    chainsToGasCheck ?? chains,
-    minGas,
-  );
+  // TODO: this should skip checks on non-evm chains
```

Comment:
Why should we skip on non-evm?

Reply:
because the scripts just throw errors for non-evm chains - so whoever runs this script for non-evm chains should be warned that there's fewer chains, but still have the option to go ahead